ci/requirements.macoswheels.in

```diff
@@ -1 +1 @@
-cibuildwheel
+cibuildwheel==1.9.0
```

Comment:
We want the `.in` file to not have versions pinned unless there is an incompatibility. Instead, keep this file as-is and run `pip-compile -U --generate-hashes --output-file=ci/requirements.macoswheels.txt ci/requirements.macoswheels.in`. That `-U` tells it to upgrade all packages.